internal/cmd/server/texts.go

```diff
@@ -13,8 +13,8 @@ func warningDeprecatedServerType(serverType *hcloud.ServerType) string {
 	}
 
 	if time.Now().After(serverType.UnavailableAfter()) {
-		return fmt.Sprintf("Attention: The server plan %q is deprecated and can no longer be ordered. Existing servers of that plan will continue to work as before and no action is required on your part. It is possible to migrate this server to another server plan by using the \"hcloud server change-type\" command.\n\n", serverType.Name)
+		return fmt.Sprintf("Attention: The Server plan %q is deprecated and can no longer be ordered. Existing servers of that plan will continue to work as before and no action is required on your part. It is possible to migrate this Server to another Server plan by using the \"hcloud server change-type\" command.\n\n", serverType.Name)
```

Comment:
Should we replace 'plan' with 'type'? This is the only place I see the use of a 'plan' for server type.